test/clj_zeppelin/core_test.clj

```diff
@@ -197,4 +198,8 @@ nth_prime_number(100)
            (delete-note @api-url))
   
 
-
+(deftest import-note-test
+  (let [notebook-url @api-url "https://raw.githubusercontent.com/hortonworks-gallery/zeppelin-notebooks/master/2DJVH9H46/note.json"]
+  (testing "import note successful or not"
```

Comment:
Add a couple more tests, e.g. netgative tests

- Test that gives a URL with no note data, should return a valid response
- instead of just testing `not nil`, retrieve the imported note . Can you check if the note matches the originally imported note?

Reply:
When the content of the imported notebook is visualized, there is no data leakage and data visualization is perfectly fine. But while the comparing the contents of imported Zeppelin notebook and the original file, they don't seem identical.